NeoML/include/NeoML/TraditionalML/KMeansClustering.h

```diff
@@ -154,6 +156,8 @@ class NEOML_API CKMeansClustering : public IClustering {
 		CDnnBlob& centers, CDnnBlob& sizes );
 	void calcClusterVariances( const CDnnBlob& data, const CDnnBlob& labels,
 		const CDnnBlob& centers, const CDnnBlob& sizes, CDnnBlob& variances );
+
+	friend struct CThreadTask;
```

Comment:
Do we need a `friend` here if it's already a nested class?

Reply:
fixed